code/game/objects/items/devices/scanners/health_analyzer.dm

```diff
@@ -592,6 +571,24 @@
 
 	diseasescan(user, patient, src)
 
+/proc/diseasescan(mob/user, mob/living/carbon/patient, obj/item/healthanalyzer/wound/scanner)
```

Comment:
Please document this proc. I know all you are doing is moving it down, but since you are touching it might aswell spruce it up. If you do I'll give you gbp for it.